### PR TITLE
locaroku to run locally

### PR DESCRIPTION
**Pull request by kk1024**
make a .env file in the client directory, run 'npm run locaroku' from the root to test locally

### CODE REVIEW

File: database/postgres.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 const Sequelize = require('sequelize');
 
-const sequelize = new Sequelize(process.env.DATABASE_URL, { dialect: 'postgres', protocol: 'postgres', dialectOptions: { ssl: true } });
+let params = {};
+if (!process.env.LOCAL) {
+  params = { dialect: 'postgres', protocol: 'postgres', dialectOptions: { ssl: true } };
+}
```
Comment (aricalves) [inline database/postgres.js]:
```
const params = Boolean(process.env.LOCAL) ? process.env.LOCAL : { dialect: 'postgres', protocol: 'postgres', dialectOptions: { ssl: true } };
```

what do you think about replacing the above lines with _this_?

Comment (aricalves) [reply database/postgres.js]: On second look, my version is really hard to parse... thoughts?

Comment (kk1024) [reply database/postgres.js]: when process.env.local is true we want params to be an empty obj, but you're setting params to equal process.env.local here? so youd replace the process.env.local after the ? with {}. if we can fit that on 1 line without airbnb getting pissed im into it, but as far as readability im not sure what is 'best' and don't really have an opinion personally
